analyzer/code.py

```diff
@@ -122,7 +122,7 @@ def run(self, path):
 
         path: Cloned repository path
         """
-        proc = subprocess.Popen(['mypy', '--ignore-missing-imports', '--allow-untyped-globals', '*.py', '**/*.py'],
+        proc = subprocess.Popen(['mypy', '--ignore-missing-imports', '--allow-untyped-globals', '.'],
```

Comment:
`python3 pyreportcard-cli.py -d .` returns below output.

```
. is not a valid Python package name
Grade: A+
Files: 29
license: 100%
mypy_lint: 100%
pep8_lint: 98%
pyflakes_lint: 99%
readme: 100%
```

The first line looks a bug (is it a bug of mypy?)

Reply:
I'll double check that. Thanks for reporting it.

Reply:
My original implementation was wrong as well. Walking the directory seems to do the trick. I also had to fix the output directory parser because the output of `mypy` does not display `./`.